README.md

```diff
@@ -576,6 +585,40 @@ sensu::backend::config_hash:
 
 The first step will not fully add the node to the cluster until the second step is performed.
 
+### Sensu backend federation
+
+Currently the federation support within this module involves configuring Etcd replicators. This allows resources to be sent from one Sensu cluster to another cluster.
+
+It's necessary that Etcd be listening on an interface that can be accessed by other Sensu backends.
+
+First configure backend Etcd to listen on an interface besides localhost:
+
+```puppet
+class { '::sensu::backend':
+  config_hash => {
+    'etcd-listen-client-urls' => ["http://localhost:2379","http://${facts['networking']['interfaces']['eth1']['ip']}:2379"],
+  },
+}
+```
+
+Next configure the Etcd replicator on the backend you wish to push resources from.
+In the following example all defined `Role` resources will be replicated to the backend at the IP address 192.168.52.30.
```

Comment:
This is a bit confusing with roles and profiles. Not sure what Role means here.

Reply:
`Role` is the Sensu Go resource, and copied from Sensu Go documentation.